src/handwritten/task.py

```diff
@@ -1007,6 +1007,8 @@ def write(self, data, auto_start=AUTO_START_UNSET, timeout=10.0):
 
         # Analog Input
         if write_chan_type == ChannelType.ANALOG_OUTPUT:
+            if number_of_samples_per_channel == 1  and number_of_channels == 1:
+                data = [data]
```

Comment:
I don't think this is right... if someone passes in a 1-element array, this would make it an array of array. Looking at the code block above this, I think you might mean `data = [element]`?

Reply:
As Brad pointed out, the change has to happen in grpc interpreter and not in `task.py`. I'll revert these changes and update with the correct solution.